frontend/src/config/cambodia/layers.json

```diff
@@ -227,7 +227,7 @@
       "styles": "rfh_16_0_300"
     },
     "chart_data": {
-      "url": "https://api.earthobservation.vam.wfp.org/stats/admin/fetch",
+      "url": "https://prism-api.ovio.org/hdc",
```

Comment:
Now that the data source is going to be the same across countries, I think it might be worth extracting into a config variable as "chart_api" or similar and make it a default.

This way you can remove "chart_data.url" from all the layers.json. But keep it available in case we need to access charts from other sources.